app/templates/index.html

```diff
@@ -104,7 +104,7 @@ <h2 id="supplier-message-heading">Sell services</h2>
           {% if logged_in %}
             <p>
               <a href="/suppliers" class="top-level-link">
-                <span>View your services and account details</span>
+                View your services and account details
```

Comment:
That's a good point. @ralph-hawkins has a design for a link for buyers too so I'll put it in and make the condition more specific.